bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -17,7 +18,13 @@ export const processForHTML5ServerOnly = fn => (message, ...args) => {
   const { envelope } = message;
   const { routing } = envelope;
 
-  const shouldSkip = routing.msgType === MSG_DIRECT_TYPE && routing.userId !== NODE_USER;
+  const selector = {
+    userId: routing.userId,
+  };
+
+  const user = Users.findOne(selector);
+
+  const shouldSkip = routing.msgType === MSG_DIRECT_TYPE && (routing.userId !== NODE_USER && user.clientType !== 'HTML5');
```

Comment:
Please also check that we have such user in db' `const shouldSkip = user && routing.msgType === MSG_DIRECT_TYPE && (routing.userId !== NODE_USER && user.clientType !== 'HTML5');`

Reply:
Hmm, are the collection always ready in this step? Like authentication/guest stuff

Reply:
They should be. This is more on a mid-meeting event, not an initializing one